src/webui/static/app/tvheadend.js

```diff
@@ -220,7 +229,7 @@ tvheadend.getContentTypeIcons = function(rec) {
   }
 
   var ret = "";
-  if (rec.new)
+  if (rec.is_new)
```

Comment:
Please, keep 'new' to keep HTTP API compatibility and use _**if ('new' in rec)**_ here...